inventory/kubejetstream/cluster.tfvars

```diff
@@ -41,6 +41,50 @@ number_of_k8s_nodes_no_floating_ip = 0
 
 flavor_k8s_node = "4"
 
+# # Uncomment when all nodes will be GPU nodes
+# # If you wish to use this var for another reason, add the ansible groups as a comma seperated list
+# # E.g "additional-group-1,additional-group2,etc"
+# supplementary_node_groups = "gpu-node"
+
+# BEGIN HYBRID CLUSTER CONFIG
+
+# # Set to true by default, but we make it explicit here
+# port_security_enabled = true
+
+# # Must be uncommented and set to 0 to use the k8s_nodes variable
+# number_of_k8s_nodes = 0
+# number_of_k8s_nodes_no_floating_ip = 0
+
+# # "<cluster-name>-k8s-node-" will be prepended to each key name and used to create the instance name.
+# # E.g the first item below would result in an instanced named "<cluster-name>-k8s-node-nf-cpu-1"
+# # For a full list of options see ./contrib/terraform/openstack/README.md#k8s_nodes
+# k8s_nodes = {
+#   "nf-cpu-1" = {
+#     "az" = "nova"
+#     "flavor": "4"
+#     "floating_ip": false
+#   },
+#   "nf-cpu-2" = {
+#     "az" = "nova"
+#     "flavor": "4"
+#     "floating_ip": false
+#   },
+#   "nf-gpu-1" = {
+#     "az" = "nova"
+#     "flavor": "10"
+#     "floating_ip": false
+#     "extra_groups": "gpu-node"
+#   },
+#   "nf-gpu-2" = {
+#     "az" = "nova"
+#     "flavor": "10"
+#     "floating_ip": false
+#     "extra_groups": "gpu-node"
```

Comment:
@ana-v-espinoza How do I create 2 profiles, 1 for CPU and 1 for GPU?
I see there is an extra group here, but it seems it is only in terraform and not in Kubernetes.
This works for GPU pods, but not for CPU:
https://github.com/zonca/jupyterhub-deploy-kubernetes-jetstream/blob/master/gpu/jupyterhub_gpu.yaml#L2-L9

Reply:
Hey Andrea,

I'm using the same "profiles" config option. Here's my snippet. You'll notice that I don't override the image in the GPU profile, as I'm using an image similar to that discussed in https://github.com/zonca/jupyterhub-deploy-kubernetes-jetstream/pull/72

```yaml
singleuser:
  image:
    name: "unidata/hybrid-gpu"
    tag: "minimal-tf"
  profileList:
  - display_name: "CPU Server"
    default: true
  - display_name: "GPU Server"
    kubespawner_override:
      extra_resource_limits:
        nvidia.com/gpu: "1"
```

Reply:
The problem is that I request a CPU server, but I spawn on a GPU node, I am wondering if it is better to restrict CPU-only users to run on CPU nodes.

Reply:
Ah okay I see what you mean! Yeah, you can taint the GPU node(s), then [add a toleration in kubespawner_override](https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html#kubespawner.KubeSpawner.tolerations) for the GPU profile.